### PR TITLE
Prepared for releasing version 0.1.5

### PR DESCRIPTION
**Pull request by otegami**
## What I did
- Added changed codes and documentation to `news.md` by PR
- Updated Datasets::VERSION from 0.1.5 to 0.1.6

## Notes for reviewers
Would you check the contents of news.md?
And I really want to add what kou did to news.md but there were a lot of changed I couldn't write.

### CODE REVIEW

File: lib/datasets/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Datasets
-  VERSION = "0.1.5"
+  VERSION = "0.1.6"
```
Comment (kou) [inline lib/datasets/version.rb:2]: Could you remove this change?
We'll do this AFTER we release `0.1.5`.

File: doc/text/news.md [hunk 2]
```diff
@@ -1,5 +1,109 @@
...
+  * `Datasets::SeabornDataList`: Added.
+    [GitHub#134][Patch by Hirokazu SUZUKI]
+
+  * `Datasets::WikipediaKyotoJapaneseEnglish`: Added.
+    [GitHub#135][Patch by abcdefg-1234567]
+
+  * Renamed Rdatasets to Rdataset.
+    [GitHub#148][Patch by Hirokazu SUZUKI]
+
+  * Removed support for Ruby 2.6. by Sutou Kouhei
```
Comment (kou) [inline doc/text/news.md:55]:
```suggestion
  * Removed support for Ruby 2.6.

  * Add missing license information.
```

**Comment by kou** [comment]
We don't need to add notes for commits that aren't related to users such as expected value update in test. Because this is for users.

**Comment by kou** [comment]
> And I really want to add what kou did to news.md but there were a lot of changed I couldn't write.

Updated.

**Comment by otegami** [comment]
> We don't need to add notes for commits that aren't related to users such as expected value update in test. Because this is for users.

Thank you so much. I've fixed: [374a79c](https://github.com/red-data-tools/red-datasets/pull/156/commits/374a79c15c661586fb869830e2017e460a30fc57)

File: lib/datasets/version.rb (hunk 1 above)
Comment (otegami) [reply lib/datasets/version.rb]: fix: [34316f1](https://github.com/red-data-tools/red-datasets/pull/156/commits/34316f166c340bf6c91175a9af14a8b78ee5189e): I'm sorry I misunderstood it.

**Comment by otegami** [comment]
Thank you for reviewing. I dealt with all of them you pointed so far🙏

**Comment by kou** [comment]
Thanks!

Could you tell me your https://rubygems.org/ account to release red-datasets gem?

**Comment by otegami** [comment]
@kou 
I've just created account. [otegami](https://rubygems.org/profiles/otegami) is my RubyGems account.

**Comment by kou** [comment]
OK.
Now, you can release red-datasets gem:

```console
$ git clone git@github.com:red-data-tools/red-datasets.git
$ cd red-datasets
$ rake release
```

**Comment by otegami** [comment]
Thank you so much I will try it now!

**Comment by otegami** [comment]
I've just released [v0.1.5](https://rubygems.org/gems/red-datasets) with @kou support.

**Pull request merged**